doc/source/reference/c-api.array.rst

```diff
@@ -1660,10 +1653,11 @@ Conversion
 .. c:function:: PyObject* PyArray_GetField( \
         PyArrayObject* self, PyArray_Descr* dtype, int offset)
 
-    Equivalent to :meth:`ndarray.getfield<numpy.ndarray.getfield>` (*self*, *dtype*, *offset*). Return
-    a new array of the given *dtype* using the data in the current
-    array at a specified *offset* in bytes. The *offset* plus the
-    itemsize of the new array type must be less than *self*
+    Equivalent to :meth:`ndarray.getfield<numpy.ndarray.getfield>` (*self*, *dtype*, *offset*). This function 
+    :meth:`steals a reference<https://stackoverflow.com/questions/23644926/what-is-reference-stealing-and-borrowing>`
+    to ``PyArray_Descr`` and return a new array of the given *dtype* 
+    using the data in the current array at a specified *offset* in bytes. 
+    The *offset* plus the itemsize of the new array type must be less than *self*
```

Comment:
Needs to be reformatted to 80 columns. You can break on any blank space. 

Missing a space between reference and `<`. The `:meth:` is not needed. Perhaps the pytohn documentation is better than a stack overflow question. `https://docs.python.org/3/c-api/intro.html?reference-count-details`

`and return a new array` -> `and returns a new array`

Reply:
Maybe I was unclear. I was suggesting
```
`steals a reference <https://docs.python.org/3/c-api/intro.html?reference-count-details>`
```
not `**steals a reference**` which will simply be rendered in bold typeface.